dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/sfdr/model/general/general/SfdrGeneralGeneral.kt

```diff
@@ -1,25 +1,20 @@
-package org.dataland.datalandbackend.model.sfdr.categories.general.subcategories
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.sfdr.model.general.general
 
-import com.fasterxml.jackson.annotation.JsonProperty
 import io.swagger.v3.oas.annotations.media.Schema
 import org.dataland.datalandbackend.model.documents.CompanyReport
-import org.dataland.datalandbackend.model.enums.commons.FiscalYearDeviation
 import org.dataland.datalandbackend.model.enums.commons.YesNoNa
 import org.dataland.datalandbackend.utils.JsonExampleFormattingConstants
 import java.time.LocalDate
 
 /**
- * --- API model ---
- * Fields of the subcategory "General" belonging to the category "General" of the sfdr framework.
-*/
+ * The data-model for the General section
+ */
 data class SfdrGeneralGeneral(
-    @field:JsonProperty(required = true)
     val dataDate: LocalDate,
 
-    @field:JsonProperty(required = true)
-    val fiscalYearDeviation: FiscalYearDeviation,
+    val fiscalYearDeviation: SfdrGeneralGeneralFiscalYearDeviationOptions,
```

Comment:
Die required true Einträge sind hier verloren gegangen.

Reply:
Confirmed:
Uploading: 
```
{
    "companyId": "e6c230d7-38e7-4db8-8d8b-a7362edca785",
    "reportingPeriod": "string",
    "data": {
        "general": {
            "general": {
                "dataDate": "2024-01-29",
                "fiscalYearDeviation": "Deviation",
                "fiscalYearEnd": "2023-01-01"
            }
        }
    }
}
```
works.

Uploading
```
{
    "companyId": "e6c230d7-38e7-4db8-8d8b-a7362edca785",
    "reportingPeriod": "string",
    "data": {
        "general": {
            "general": {
                "dataDate": "2024-01-29",
                "fiscalYearEnd": "2023-01-01"
            }
        }
    }
}
```

Results in a 400 error (value failed for JSON property fiscalYearDeviation due to missing (therefore NULL) value for creator parameter fiscalYearDeviation which is a non-nullable type)